.circleci/config.yml

```diff
@@ -29,7 +29,7 @@ workflows:
           matrix:
             parameters:
               # Run with MSRV and some modern stable Rust
-              rust-version: ["1.56.1", "1.60.0"]
+              rust-version: ["1.56.1", "1.62.0"]
```

Comment:
What's the point of keeping `1.56.1`?

Reply:
We try to keep the range of supported compilers as wide as practically possible in order to reduce the friction for our users and the Rust compiler related support requests low. 1.56.1 us used as the compiler for all CI jobs. We have 1.62.0 on top in case of clippy to also test the codebase with the latest clipy rules.

I think we could bump 1.56.1 a bit these days but have no real need to.